include/pybind11/pybind11.h

```diff
@@ -52,6 +52,38 @@ PYBIND11_WARNING_DISABLE_MSVC(4127)
 
 PYBIND11_NAMESPACE_BEGIN(detail)
 
+constexpr const char *whitespaces = " \t\n\r\f\v";
```

Comment:
Could you please move this inside the function?

If it needs to stay outside, we'd need a more specific name (long is fine), but generally it works best in my experience to keep such variables in the context in which they are needed, and pull them out only if they are actually needed elsewhere.